keycloak/themes/hmda/login/register.ftl

```diff
@@ -171,6 +171,11 @@ $(document).ready(function() {
     getInstitutions(emailToDomain($('#email').val()));
   }
 
+  // remove whitespace from email to prevent 'invalid email'
+  $('#email').on('blur', function(e) {
```

Comment:
Can you cache the `$('#email')` bit here so we don't have to touch the DOM so much`?

`var email = $('#email')`